src/Avatar/__stories__/Avatar.stories.js

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable react/no-multi-comp */
+import Avatar from '../Avatar';
+import React from 'react';
+
+export default {
+    title: 'Component API/Avatar',
+    component: Avatar
+};
+
+export const primary = () => (<Avatar glyph='cart' size='m' />);
+
+export const sizes = () => (
+    <div className='fr-container'>
```

Comment:
This got changed out from underneath you - but this class is now `fddocs-container` - that will get you the spacing you want.